lightningd/opening_control.c

```diff
@@ -371,6 +371,10 @@ static void opening_funder_finished(struct subd *openingd, const u8 *resp,
 	/* Extract the change output and add it to the DB */
 	wallet_extract_owned_outputs(ld->wallet, fundingtx, NULL, &change_satoshi);
 
+	/* Make sure we recognize our change output by its scriptpubkey in future */
+	if (tal_count(fundingtx->output) == 2)
```

Comment:
Would not work if ever we make a `multifundchannel`, but maybe cross that bridge when we get there.

Reply:
Good point, I'll add a `FIXME`, presumably we'd have to change things around anyway, since the code explicitly expects a single funding output in its current state.